src/search_file.c

```diff
@@ -166,7 +166,10 @@ static long get_bundle_size(const char *bundle)
 		return b->size;
 	}
 
-	b = calloc(1, sizeof(struct bundle_size));
+	b = (struct bundle_size*)calloc(1, sizeof(struct bundle_size));
```

Comment:
we don't need to cast void*, so it's fine to do:

     	b = calloc(1, sizeof(struct bundle_size));

Reply:
nice, I was unsure